code/modules/uplink/uplink_items.dm

```diff
@@ -1807,13 +1807,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	surplus = 0
 	include_modes = list(/datum/game_mode/nuclear)
 
-/datum/uplink_item/implants/mantis_kit
-	name = "G.O.R.L.E.X.. Mantis Blades Kit"
-	desc = "Comes with 2 G.O.R.L.E.X. Mantis blades. All packaged with autosurgeons."
-	item = /obj/item/storage/briefcase/syndie_mantis
-	cost = 16
-	surplus = 0
-	exclude_modes = list(/datum/game_mode/infiltration) // yogs: infiltration
+/datum/uplink_item/implants/mantis
+    name = "G.O.R.L.E.X. Mantis Blade"
+    desc = "One G.O.R.L.E.X Mantis blade implant able to be retracted inside your body at will for easy storage and concealing, 2 blades can be used at once."
+    item = /obj/item/autosurgeon/organ/syndicate/syndie_mantis
+    cost = 7
+    surplus = 0
+    exclude_modes = list(/datum/game_mode/infiltration) // yogs: infiltration
```

Comment:
edit these so they are tabs instead of spaces

Reply:
@SomeguyManperson  would if i knew how i didnt make this code im just PRing it. i also dont know what you want me to tab here?

Reply:
figured out what you want me to change and did just gotta figure out how to put it here